test/sql/function/list/list_contains.test

```diff
@@ -0,0 +1,190 @@
+# name: test/sql/function/list/list_contains.test
+# description: Test list_ccontains function
+# group: [list]
+
+statement ok
+PRAGMA enable_verification
+
+statement ok
+create table TEST (i int[]);
+
+statement ok
+insert into TEST values ([2,1,3]), ([2,3,4]), ([1]);
```

Comment:
Can we also get a test case with a non-constant value, e.g.:
```sql
statement ok
create table TEST2 (i int[], int);

statement ok
insert into TEST2 values ([2,1,3], 2), ([2,3,4], 5), ([1], NULL);

query T
select list_contains(i, j) from TEST2
----
1
0
NULL
```

I suspect this does not work yet